bftengine/src/bcstatetransfer/BCStateTran.hpp

```diff
@@ -187,6 +187,11 @@ class BCStateTran : public IStateTransfer {
   // TODO - the next function should be removed or refactored into special dedicated class
   void setReconfigurationEngineImpl(std::shared_ptr<ClientReconfigurationEngine> cre);
 
+  static void stateTransferMsgDeleter(BCStateTranBaseMsg* m) {
```

Comment:
Please move to cpp. It's not a one liner.
It also doesn look like the right place to put it in the header (mo that the header is well orgnized anyway...)